properties.ini

```diff
@@ -0,0 +1,170 @@
+; this file is used to define all necessary options for virt-who testing
+; all default values are null, need to update according to the test requirements
+
+[host]
+; the rhel system to run virt-who service
+; virtwho/docker.py and kickstart.py can help to install a host
+ip=
+user=
+passwd=
+
+[repo]
+; the base url of the rhel compose for special cases
+rhel_base=
+
+[proxy]
+; a proxy server is necessary if run the related test cases
+server=
+port=
+
+[rhsm]
+; the datas of stage or product candlepin server
+server=
+user=
+passwd=
+org=
+
+[satellite]
+; the datas of satellite server
+; virtwho/satellite.py can help to deploy a satellite server
+server=
+admin_user=
+admin_passwd=
+default_org=
+default_env=
+extra_org=
+activation_key=
```

Comment:
To match the configure options from /etc/virt-who.d, 
#rhsm_hostname=
#rhsm_port=
#rhsm_username=
#rhsm_password=
#rhsm_encrypted_password=
#rhsm_prefix=/rhsm

how about to add port and prefix to this section? meanwhile, it's more better if we change the user and passwd to a full name: username and password
[rhsm]
server=
username=
password=
prefix=
port=
default_org=

[satellite]
server=
username=
password=
prefix=
port=
default_org=
secondary_org=
activation_key=

Think about the ssh account for satellite, in our test cases, the ssh is necessary or not for satellite if we use the API to connect

Reply:
yeah, forgot the prefix and port it's a mistake.
it's also good to use all full name and to match the virt-who config files